explore/src/main/scala/explore/cache/ProgramCacheController.scala

```diff
@@ -354,12 +365,16 @@ object ProgramCacheController
           val updateConfigurationRequests
             : Resource[IO, Stream[IO, ProgramSummaries => ProgramSummaries]] =
             ProgramQueriesGQL.ConfigurationRequestSubscription
-              .subscribe[IO](ConfigurationRequestEditInput(props.programId.assign))
+              .subscribe[IO](ConfigurationRequestEditInput(props.programId.assign))(using
+                summon,
```

Comment:
Can the  `summon` be omitted?

Reply:
I'll try again, but I don't think so. I think you need something in every position of the using parameter list. The existing subscriptions specifying a error policy are doing this. But, I guess I could maybe just do a `given` prior to the subscription,

Reply:
I was under the impression that Scala 3 was now smart enough to type-match givens in case some were not declared. If that's not the case, just leave the `summon`.

Reply:
I thought so true, so I tried that earlier. It may `supposed to be` that smart, but apparently it is not.